### PR TITLE
Fix #4439: Change speaker icon to headset icon

### LINKED ISSUE #4439: Change speaker icon to headphones icon
**Is your feature request related to a problem? Please describe.**

According to the user reports, some users do not understand what the speaker button on the exploration screen does. Change it to a headphones button instead. Make sure that the button has both on and off states displayed correctly. This issue is a part of the GSoC Interactive Onboarding Flow project.

### PR DESCRIPTION
**Pull request by JishnuGoyal**
## Explanation

  This PR fixes #4439 by adding new drawable resources (icons) -- headset on and headset off. This PR is a part of the GSoC project: Interactive Onboarding Flow.
## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

Using Pixel 4 API 28 Emulator:

Before:
image

image

image

image

After: 

Voice-over off portrait
image

Voice-over on portrait
image

Voice-over on landscape
image

Voice-over off landscape
image

### CODE REVIEW

**Comment by JishnuGoyal** [comment]
image

This is why I changed from color/white to #FFFFFF

**Comment by JishnuGoyal** [comment]
@BenHenning would it be possible to re-run the CI? It says ktlint file is currupted. I didn't commit any changes to this file but I re-downloaded it after CI fail.

**Comment by JishnuGoyal** [comment]
PTAL @BenHenning

**Review by BenHenning — APPROVED** [review]
Thanks @JishnuGoyal generally LGTM, but could also please add the 'before' screenshots to show the old icon, as well?

**Comment by BenHenning** [comment]
Sending over to @rt4914 as well for codeowners.

File: app/src/main/res/drawable/ic_baseline_headset_24.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<vector android:autoMirrored="true" android:height="24dp"
+    android:tint="#FFFFFF" android:viewportHeight="24"
+    android:viewportWidth="24" android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <path android:fillColor="#FFFFFF" android:pathData="M12,1c-4.97,0 -9,4.03 -9,9v7c0,1.66 1.34,3 3,3h3v-8H5v-2c0,-3.87 3.13,-7 7,-7s7,3.13 7,7v2h-4v8h3c1.66,0 3,-1.34 3,-3v-7c0,-4.97 -4.03,-9 -9,-9z"/>
+</vector>
```
Comment (rt4914) [inline app/src/main/res/drawable/ic_baseline_headset_24.xml:11]: Please reformat this file too.

Press `Control + Alt + L` - reformatting shortcut to fix the formatting.

**Review by rt4914 — APPROVED** [review]
LGTM, Thanks.

File: app/src/main/res/drawable/ic_baseline_headset_24.xml (hunk 1 above)
Comment (JishnuGoyal) [reply app/src/main/res/drawable/ic_baseline_headset_24.xml:11]: done, thanks!

**Comment by JishnuGoyal** [comment]
PTAL @BenHenning

File: app/src/main/res/drawable/ic_baseline_headset_24.xml (hunk 1 above)
Comment (BenHenning) [reply app/src/main/res/drawable/ic_baseline_headset_24.xml:11]: This seems addressed in the latest commit--marking this as resolved now.

**Review by BenHenning — APPROVED** [review]
Thanks @JishnuGoyal! Everything still LGTM. Going and merging this.

Thanks also for the review @rt4914!

**Pull request merged**